arcor2_AREditor/Assets/TABLET/Scripts/LeftMenu.cs

```diff
@@ -365,8 +379,34 @@ public abstract class LeftMenu : MonoBehaviour {
         }
     }
 
+    public async void ModelPositioningButtonClick() {
+        if (!SceneManager.Instance.SceneStarted) {
+            Notifications.Instance.ShowNotification("Failed to open model manipulation menu", "Scene offline");
+            return;
+        } else if (!SceneManager.Instance.IsRobotAndEESelected()) {
+            OpenRobotSelector(ModelPositioningButtonClick);
+            return;
+        }
 
-
+        if (ModelPositioningMenu.Instance.isActiveAndEnabled) {
+            await ModelPositioningMenu.Instance.TurnOff();
+            SelectorMenu.Instance.gameObject.SetActive(true);
+            //ModelSteppingMenu.Instance.ExitDialogShow();
```

Comment:
remove it or add comment - after some time, no one will know why it is there :-)

Reply:
I added some comments, check if they're sufficient, please.